### DISCUSSION TITLE
ghostwriter

### DISCUSSION THREAD

**Discussion by bagelwaffle** [catppuccin/catppuccin#2057] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

[Ghostwriter](https://ghostwriter.kde.org/) can be themed through the app itself.
![ghostwriter-app-themeing](https://github.com/catppuccin/catppuccin/assets/134188288/d701786e-63a2-4bad-b1ae-449e3515811d)
The colours set here is saved to a json file.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by Sourcastic** [comment]
Hm, is it possible to add the .json files to ghostwriter? I tried to do this back when ghostwriter had just been released, and I can't recall any way I found within the documentation to share the themes.

**Reply by bagelwaffle** [reply]
I can confirm that you can add the .json files to ghostwriter.
As a flatpak user, it puts my themes in `~/.var/app/io.github.wereturtle.ghostwriter/config/ghostwriter/themes`
For people that install through the package manager it should save in `~/.config/ghostwriter/themes`

After moving the theme json to the folder, a restart of the application is needed

Here is a ghostwriter theme - [Dracula](https://draculatheme.com/ghostwriter)

**Reply by Sourcastic** [reply]
@bagelwaffle Ah, I see. Well, the best approach to get this added as a port would be to make a repo following [the port creation guide.](https://github.com/catppuccin/catppuccin/blob/main/docs/port-creation.md) Otherwise, you could wait for someone to pick this up. I'd be willing to work on this whenever I get free time personally.

**Comment by bagelwaffle** [comment]
I’ll work on the port when I get home, I’ve already made a theme with latte for light mode, mocha for dark, and blue as accent colour, all I have to do now is change the accent colours and change the other colours to match the different themes :)


May 30, 2023 at 08:26 by ***@***.***:

**Comment by bagelwaffle** [comment]
Made a few more themes :)
![cat](https://github.com/catppuccin/catppuccin/assets/134188288/a9b003ad-0aed-4260-8848-07b31c748c0c)

All I have to do now is make it for all the different accent colours, should be fairly easy with puccinier from the catppuccin toolbox

**Comment by bagelwaffle** [comment]
The port is now in port review :)

[Repo](https://github.com/bagelwaffle/ghostwriter)
 
#2062
